src/ooxml/java/org/apache/poi/xddf/usermodel/chart/XDDFChart.java

```diff
@@ -712,10 +734,29 @@ public CellReference setSheetTitle(String title, int column) {
         XSSFRow row = this.getRow(sheet, 0);
         XSSFCell cell = this.getCell(row, column);
         cell.setCellValue(title);
-        this.updateSheetTable(sheet.getTables().get(0).getCTTable(), title, column);
+        
```

Comment:
if sheet doesn;t have table object then first create it and then pass newly created table to update sheet data. so created a method GetSheetTable to return table object.